MyBus/MyBus/MainViewController.swift

```diff
@@ -207,22 +207,40 @@ class MainViewController: UIViewController {
             self.mapViewController.mapView.showsUserLocation = true
             // Convert tap location (CGPoint) to geographic coordinates (CLLocationCoordinate2D)
             let tappedLocation = self.mapViewController.mapView.convertPoint(tap.locationInView(self.mapViewController.mapView), toCoordinateFromView: self.mapViewController.mapView)
-
+            
             progressNotification.showLoadingNotification(self.view)
-
-            Connectivity.sharedInstance.getAddressFromCoordinate(tappedLocation.latitude, longitude: tappedLocation.longitude) { (routePoint, error) in
-                if let destination = routePoint {
-                    if let _ = self.mapViewModel.origin {
-                        self.newDestination(destination)
-                    } else {
-                        self.newOrigin(destination)
-                    }
-                    self.homeNavigationBar(self.mapViewModel)
+            
+            if let annotations = self.mapViewController.mapView.annotations {
+                if( annotations.count > 1 ){
+                    self.progressNotification.stopLoadingNotification(self.view)
+                    let alert = UIAlertController(title: "Borrando busqueda actual", message: "Está seguro que quiere borrar la busqueda actual?", preferredStyle: UIAlertControllerStyle.ActionSheet)
```

Comment:
Messages text must be in Localizable strings file. Please don't close current PR to do changes, just commit & push 👍